android/build.gradle

```diff
@@ -59,7 +59,6 @@ android {
 }
 
 dependencies {
-  compile 'com.facebook.react:react-native:+'
-  compile "com.google.android.gms:play-services-base:9.4.0"
-  compile 'com.google.android.gms:play-services-maps:9.4.0'
+    compile 'com.facebook.react:react-native:+'
+    compile 'com.google.android.gms:play-services:9.6.1'
```

Comment:
9.6 is needed for the map styling https://developers.google.com/android/guides/releases